docs/en/alab/project_configuration.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: docs
+comment: no
+header: true
+seotitle: Annotation Lab | John Snow Labs
+title: Configuration
+permalink: /docs/en/alab/project_configuration
+key: docs-training
+modify_date: "2022-10-13"
+use_language_switcher: "Python-Scala"
+show_nav: true
+sidebar:
+    nav: annotation-lab
+---
+
+We currently support multiple predefined project configurations. The most popular ones are Text Classification, Named Entity Recognition (NER) and  Visual NER . Create a setup from scratch or customize a predefined one according to your needs.
+
+For customizing a predefined configuration, click on the corresponding link in the table above and then navigate to the Labeling configuration tab and manually edit or update it to contain the labels you need.
+
+After you finish editing the labels you want to define for your project click the “Save” button.
+
+### Project templates 
+We currently support multiple predefined project configurations. The most popular ones are **Text Classification** and **Named Entity Recognition**. Create a setup from scratch or customize a predefined one according to your needs.
+
+
+ <img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/template.jpg" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
```

Comment:
You can remove the `align:center;` from the style attribute as it is incorrect property field. And instead of `box-shadow: ....` you can add the class `image__shadow` to the class attribute.